Tests/scripts/utils/tests/test_get_modified_files_for_testing.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from Tests.scripts.collect_tests_and_content_packs import get_modified_files_for_testing, COMMON_YML_LIST
+
+
+class TestGetModifiedFilesForTesting:
+    """"
+    Given: A git-diff output.
+
+    When: Collecting tests
+
+    Then: Validate the output contains or not the given files
+    """
+
+    def test_python_file(self, mocker):
+        diff_line = "M       Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py"
+        yml_file = "Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.yml"
+        mocker.patch(
+            "Tests.scripts.collect_tests_and_content_packs.glob.glob",
+            return_value=[yml_file])
+        assert ([yml_file], [], [], False, [], set(), False, False) == get_modified_files_for_testing(diff_line)
```

Comment:
can we make it a bit more indicative what exactly we're asserting here?
its hard to understand what are all these `[]` and booleans

Reply:
you're right - So as the function itself. I'll figure it out